index.js

```diff
@@ -40,8 +40,12 @@ function pinoStackdriver (line) {
 
 const transform = split(pinoStackdriver)
 
-function createStream () {
-  return pumpify(transform, process.stdout)
+function createStream (dest) {
```

Comment:
```suggestion
function createStream (dest = process.stdout) {
```
(instead of logic below)